legend-engine-testable/src/main/test/java/org/finos/legend/engine/test/runner/shared/assertion/TestTestAssertionEvaluator.java

```diff
@@ -12,7 +12,7 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-package org.finos.legend.engine.testable.service.assertion;
+package org.finos.legend.engine.test.runner.shared.assertion;
 
```

Comment:
Can we move this to 
org.finos.legend.engine.testable.assertion

Reply:
Package still seems wrong

Reply:
seems like the old change, the package is different in the actual changes